apps/dashboard/src/components/contract-functions/interactive-abi-function.tsx

```diff
@@ -352,22 +424,32 @@ export const InteractiveAbiFunction: React.FC<InteractiveAbiFunctionProps> = ({
               rightIcon={<Icon as={FiPlay} />}
               colorScheme="primary"
               isLoading={readLoading}
-              type="submit"
+              onClick={handleContractRead}
               form={formId}
             >
               Run
             </Button>
           ) : (
-            <TransactionButton
-              isDisabled={!abiFunction}
-              colorScheme="primary"
-              transactionCount={1}
-              isLoading={mutationLoading}
-              type="submit"
-              form={formId}
-            >
-              Execute
-            </TransactionButton>
+            <>
+              <ToolTipLabel label="Simulate the transaction to see its potential outcome without actually sending it to the network. This action doesn't cost gas.">
```

Comment:
its a bit weird to get the tooltip everytime i hover the button, can we do a little `i` icon instead?